canton/BUILD.bazel

```diff
@@ -50,3 +50,11 @@ java_import(
     jars = [":canton-lib.jar"],
     visibility = ["//visibility:public"],
 )
+
+java_binary(
+    name = "canton-ee",
+    #main_class = "com.digitalasset.canton.CantonEnterpriseApp",
+    main_class = "com.digitalasset.canton.CantonCommunityApp",
+    visibility = ["//ledger-service:__subpackages__"],
+    runtime_deps = ["//canton:lib"],
+)
```

Comment:
@garyverhaegen-da Here we need to have the canton EE jar instead of the CE one. 
I think the last release version is enough, we probably do not need the daily snapshot.

Reply:
done as part of #17039